src/diagnostics/wizardSteps/StartDebugAdapterStep.ts

```diff
@@ -20,7 +20,8 @@ export class StartDebugAdapterStep extends WizardStep {
 
             vscode.commands.executeCommand('workbench.view.debug');
         });
-        await vscode.debug.startDebugging(vscode.workspace.workspaceFolders[0], {
+        const folder = vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0] : undefined;
```

Comment:
See https://github.com/Microsoft/vscode/issues/37559 for more details. 

The code was that way because 'undefined' didn't work before. If you found out  'undefined' can work now, it means the above bug fix has been released. Then we should always use ‘undefined’. 

`vscode.workspace.workspaceFolders[0]` is arbitrary and does not really have any significance in our business.

Reply:
makes sense.  Fixed now.